### ISSUE TITLE
Calendar screen is just cooked

### ISSUE THREAD

**Issue by Sp33dyB01** [Flex936/anistream#97]
Cannot go to navbar
doesn't scroll up fully
when the feed is scrolled to the right and scrolled to the left it doesnt appear correctly
you can scroll to the next row from another row
